src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
@agilob If I understand issue topic right, you don't want to inherit parent stile right? What you need is set the flag Qt::WA_WindowPropagation. Just change to this:

```c++
QString name = QInputDialog::getText(this, title, title + ":", QLineEditNormal, QString(), 0, Qt::WA_WindowPropagation);
// also check this window's flags, please
```

Might be Qt bug - let me check